boards/tenstorrent/tt_blackhole/tt_blackhole_dmc.dtsi

```diff
@@ -39,6 +39,12 @@
 			gpios = <&gpiob 5 GPIO_ACTIVE_LOW>;
 		};
 
+		smba {
```

Comment:
Where is this being used?

Reply:
Nowhere currently.

Reply:
Fixed under the assumption that there is no way to use `&smba` instead of `int-gpios = <&gpioa 1 GPIO_ACTIVE_LOW>`.